go-controller/pkg/ovn/base_network_controller_secondary.go

```diff
@@ -295,6 +295,21 @@ func (bsnc *BaseSecondaryNetworkController) addLogicalPortToNetworkForNAD(pod *k
 	var ops []ovsdb.Operation
 	var lsp *nbdb.LogicalSwitchPort
 	var newlyCreated bool
+	var lspOpts []nbdb.LSPOption
+	var kubevirtLiveMigrationStatus *kubevirt.LiveMigrationStatus
+
+	if kubevirt.IsPodAllowedForMigration(pod, bsnc.NetInfo) {
+		kubevirtLiveMigrationStatus, err = kubevirt.DiscoverLiveMigrationStatus(bsnc.watchFactory, pod)
+		if err != nil {
+			return fmt.Errorf("failed to discover Live-migration status: %w", err)
+		}
+	}
+
+	// actions on the pods' LSP are only triggerred from the target pod
+	shouldCheckLiveMigration := kubevirtLiveMigrationStatus != nil && pod.Name == kubevirtLiveMigrationStatus.TargetPod.Name
+	if shouldCheckLiveMigration {
+		lspOpts = append(lspOpts, nbdb.WithLogicalSwitchPortEnabled(kubevirtLiveMigrationStatus.State == kubevirt.LiveMigrationTargetDomainReady))
```

Comment:
Add a comment explaining why this cannot be done at a separate opt and we should mark the LSP as Enabled field before creation.